src/components/Header/Header.jsx

```diff
@@ -1,11 +1,16 @@
+import { useContext } from "react";
 import Navbar from '../Navbar/Navbar';
+import { AuthContext } from "../../containers/AuthContext";
 // import styles from './Header.module.scss';
 
 export default function Header() {
+    const authContext = useContext(AuthContext);
+    console.log(authContext.token);
     return (
         <header>
             <div className="container">
                 <p>Header </p>
+                {(authContext.token)?<p>Hello, {authContext.token}</p>:''}
```

Comment:
looks like something temporary, 
do we really need it here?

Reply:
yes, it was temporary  to check that we have access to authContext.token. - I deleted this code